game_eggs/vintage_story/egg-vintage-story.json

```diff
@@ -17,38 +22,51 @@
     },
     "scripts": {
         "installation": {
-            "script": "#!\/bin\/ash\r\n# Vintage Story Installation Script\r\n#\r\n# Server Files: \/mnt\/server\r\napt update\r\napt install -y screen wget curl jq\r\n\r\n# Fetching latest stable server release\r\nif [ -z \"${RELEASE_VERSION}\" ] || [ \"${RELEASE_VERSION}\" == \"latest\" ]; then\r\n    DOWNLOAD_URL=`curl -s 'http:\/\/api.vintagestory.at\/stable.json' | jq -r '[.[]] | .[0].server.urls.cdn'`;\r\n# Use specified server release\r\nelse\r\n    DOWNLOAD_URL=https:\/\/cdn.vintagestory.at\/gamefiles\/stable\/vs_server_${RELEASE_VERSION}.tar.gz\r\nfi\r\n\r\ncd \/mnt\/server\/\r\n\r\ncurl -o vs_server.tar.gz ${DOWNLOAD_URL}\r\ntar -xzf vs_server.tar.gz\r\n\r\nrm vs_server.tar.gz",
+            "script": "#!\/bin\/bash\r\n# Vintage Story Installation Script\r\n#\r\n# Server Files: \/mnt\/server\r\nDEBIAN_FRONTEND=noninteractive apt update\r\nDEBIAN_FRONTEND=noninteractive apt install -y screen wget curl jq\r\n\r\nBRANCH=\"${FILES_BRANCH}\"\r\n\r\n# Fetching latest stable server release\r\nif [ -z \"${RELEASE_VERSION}\" ] || [ \"${RELEASE_VERSION}\" == \"latest\" ]; then\r\n    case \"${BRANCH}\" in\r\n        \"stable\")\r\n            API_URL=\"http:\/\/api.vintagestory.at\/stable.json\"\r\n            ;;\r\n        \"unstable\")\r\n            API_URL=\"http:\/\/api.vintagestory.at\/unstable.json\"\r\n            ;;\r\n        \"pre\")\r\n            API_URL=\"http:\/\/api.vintagestory.at\/pre.json\"\r\n            ;;\r\n        *)\r\n            echo \"Invalid stage; fallback to stable\"\r\n            API_URL=\"http:\/\/api.vintagestory.at\/stable.json\"\r\n            ;;\r\n    esac\r\n    DOWNLOAD_URL=`curl -s \"$API_URL\" | jq -r 'if ([.[]] | .[0].server.urls.cdn) != null then [.[]] | .[0].server.urls.cdn else [.[]] | .[0].server.urls.local end'`;\r\n# Use specified server release\r\nelse\r\n    case \"${BRANCH}\" in\r\n    \"stable\")\r\n        DOWNLOAD_URL=https:\/\/cdn.vintagestory.at\/gamefiles\/stable\/vs_server_${RELEASE_VERSION}.tar.gz\r\n        ;;\r\n    \"unstable\")\r\n        DOWNLOAD_URL=https:\/\/account.vintagestory.at\/files\/unstable\/vs_server_${RELEASE_VERSION}.tar.gz\r\n        ;;\r\n    \"pre\")\r\n        DOWNLOAD_URL=https:\/\/cdn.vintagestory.at\/gamefiles\/pre\/vs_server_${RELEASE_VERSION}.tar.gz\r\n        ;;\r\n    *)\r\n        echo \"Invalid stage; fallback to stable\"\r\n        DOWNLOAD_URL=https:\/\/cdn.vintagestory.at\/gamefiles\/stable\/vs_server_${RELEASE_VERSION}.tar.gz\r\n        ;;\r\n    esac\r\nfi\r\n\r\ncd \/mnt\/server\/\r\n\r\ncurl -o vs_server.tar.gz ${DOWNLOAD_URL}\r\ntar -xzf vs_server.tar.gz\r\n\r\nrm vs_server.tar.gz",
```

Comment:
You dont set debian to non interactive here? And you should not need screen

Reply:
Screen was previously in the egg, due to [official install instructions](https://wiki.vintagestory.at/index.php?title=Setting_up_a_Multiplayer_Server#Requirements_for_Ubuntu_(Debian,_Mint,_...)); will remove here.

Reply:
`DEBIAN_FRONTEND=noninteractive` was purely added to suppress few errors, eg: `debconf: unable to initialize frontend`; mistakenly left it at `apt update` command :/